plugins/utils/which-key.nix

```
@@ -7,23 +7,146 @@
   ...
 }:
 with lib;
-{
-  options.plugins.which-key = {
-    enable = mkEnableOption "which-key.nvim, a plugin that popup with possible key bindings of the command you started typing";
+let
+  opt = options.plugins.which-key;
```

```suggestion
  inherit (lib) types;
  inherit (lib.nixvim) defaultNullOpts;

  opt = options.plugins.which-key;
```

This would allow removing `helpers` from the module args.